nix/proto3-suite.nix

```diff
@@ -1,22 +1,24 @@
 { mkDerivation, aeson, aeson-pretty, attoparsec, base
 , base64-bytestring, binary, bytestring, cereal, containers
 , contravariant, deepseq, doctest, fetchgit, filepath, foldl
-, hashable, haskell-src, insert-ordered-containers, lens, mtl
-, neat-interpolation, optparse-applicative, optparse-generic
-, parsec, parsers, pretty, pretty-show, proto3-wire, QuickCheck
-, quickcheck-instances, range-set-list, safe, semigroups, stdenv
-, swagger2, system-filepath, tasty, tasty-hunit, tasty-quickcheck
-, text, transformers, turtle, vector
+, generic-arbitrary, hashable, haskell-src
+, insert-ordered-containers, lens, mtl, neat-interpolation
+, optparse-applicative, optparse-generic, parsec, parsers, pretty
+, pretty-show, proto3-wire, QuickCheck, quickcheck-instances
+, range-set-list, safe, semigroups, stdenv, swagger2
+, system-filepath, tasty, tasty-hunit, tasty-quickcheck, text
+, transformers, turtle, vector
 }:
 mkDerivation {
   pname = "proto3-suite";
   version = "0.4.0.0";
   src = fetchgit {
-    url = "https://github.com/awakesecurity/proto3-suite.git";
-    sha256 = "091db048hgcq5idvf5gaiqb6hzbs7g1dz6xjqdx61dw2yxgdm957";
-    rev = "973c317b91405a11438e3a21706024bfa3d754df";
+    url = "https://github.com/awakesecurity/proto3-suite";
+    sha256 = "0wyw8601k6rm8wjz8mf6pl43ccm2nmikrd6ddr7h3qj0aj55qjyc";
+    rev = "f5ca2bee361d518de5c60b9d05d0f54c5d2f22af";
     fetchSubmodules = true;
   };
+  jailbreak = true;
```

Comment:
Necessary for QuickCheck 2.13. See https://github.com/awakesecurity/proto3-suite/pull/114

Reply:
Bumped the rev and removed `jailbreak = true` now that the PR has been merged.

Reply:
Oh, nevermind my comment then :)